testing/tests/DevExpress.ui.widgets.form/form.markup.tests.js

```diff
@@ -773,6 +776,210 @@ QUnit.module('Form', () => {
     });
 });
 
+QUnit.module('Class check', ()=>{
+    ['dxCheckBox', 'dxSwitch', 'dxRadioGroup'].forEach(editorType => {
+        const componentName = editorType.split('dx')[1].toLowerCase();
+
+        test(`${editorType} should have a css class ${TOGGLE_CONTROLS_PADDING_CLASS} when labelLocation=top, label.alignment=default (default=left), label.visible=true (T1126956)`, function(assert) {
+            const $form = $('#form').dxForm({
+                labelLocation: 'top',
+                items: [
+                    {
+                        itemType: 'group',
+                        items: [
+                            {
+                                itemType: 'group',
+                                items: [{
+                                    dataField: editorType,
+                                    label: { visible: true },
+                                    editorType,
+                                }],
+                            },
+                        ],
+                    },
+
```

Comment:
Let's remove excess linebreaks in all tests

Reply:
fixed